static/src/stylesheets/module/crosswords/_clues.scss

```diff
@@ -14,7 +14,7 @@
 .crossword__clue {
     @include fs-textSans(4);
     position: relative;
-    padding: $gs-baseline * .25 $gs-baseline;
+    padding: $gs-baseline * .25 $gs-baseline $gs-baseline * .25 0;
```

Comment:
The new design has clues left-aligned with the heading, this removes left padding.